src/main/java/org/folio/circulation/resources/RequestScheduledNoticeProcessingResource.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulation.resources;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.MultipleRecords;
+import org.folio.circulation.domain.notice.schedule.RequestScheduledNoticeHandler;
+import org.folio.circulation.domain.notice.schedule.ScheduledNotice;
+import org.folio.circulation.domain.notice.schedule.ScheduledNoticesRepository;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.http.HttpClient;
+
+public class RequestScheduledNoticeProcessingResource extends ScheduledNoticeProcessingResource {
+
+  private static final int SCHEDULED_NOTICES_PROCESSING_LIMIT = 100;
+
+  public RequestScheduledNoticeProcessingResource(HttpClient client) {
+    super("/circulation/request-scheduled-notices-processing" ,client);
+  }
+
+  @Override
+  protected CompletableFuture<Result<MultipleRecords<ScheduledNotice>>> findNoticesToSend(
+    ScheduledNoticesRepository scheduledNoticesRepository) {
+
+    final DateTime systemTime = DateTime.now(DateTimeZone.UTC);
+    return scheduledNoticesRepository.findRequestNoticesToSend(systemTime, SCHEDULED_NOTICES_PROCESSING_LIMIT);
```

Comment:
Is it necessary to use local var here?
`final DateTime systemTime = DateTime.now(DateTimeZone.UTC);`
I would recommend to use single statement 
```
 return scheduledNoticesRepository
      .findRequestNoticesToSend(DateTime.now(DateTimeZone.UTC), SCHEDULED_NOTICES_PROCESSING_LIMIT);
```

Reply:
replaced by a single statement